src/selectedtests/app/app.py

```diff
@@ -0,0 +1,34 @@
+"""
+Application to serve API of selected-tests service.
+"""
+
+from flask import Flask
+
+DEFAULT_PORT = 8080
+
+
+def create_app() -> Flask:
+    """
+    Create an instance of the flask application.
+
+    :return: Instance of flask application.
+    """
+    app = Flask(__name__)
+
+    @app.route("/health")
+    def health():
+        """
+       Get information about whether service is running
+       """
+        return "ok"
```

Comment:
We might consider returning json here. That would make it easy to expand in the future.